### PR TITLE
worker thread approach + train_troops + fixes

### PR DESCRIPTION
**Pull request by pkeweloh**
worker thread improvements and the train_troops feature.
fixed some issues with selenium when clicking the inputs.

### CODE REVIEW

File: src/worker.py
Comment (breuerfelix) [inline src/worker.py]: I think we don't need this file anymore right ? :)

File: start.py
Comment (breuerfelix) [inline start.py]: maybe remove the debug = true

File: start.py
Comment (breuerfelix) [inline start.py]: could you also add some lines about these features in the readme maybe ? i think most people are looking there if they wanna know what functions there are :D

you could just copy paste what i wrote for some other functions and change parameters, nothing fancy ! just so people know this features exsits :)

File: src/robber_hideouts.py
Comment (breuerfelix) [inline src/robber_hideouts.py]: you can just remove it if we dont need it anymore :)

**Review by breuerfelix — CHANGES_REQUESTED** [review]
just some minor changes ! love this PR, i will merge it after these changes :)

really awesome features, thanks for the time you put into this !

btw i will also remove the 'deprecated' in the readme after merging this PR cause i didnt realise this bot actually still works :D

File: start.py
Comment (pkeweloh) [reply start.py]: sure, I will prepare the texts!

File: src/robber_hideouts.py
Comment (pkeweloh) [reply src/robber_hideouts.py]: All right, I'd commented it out in first place because for me it didn't make sense reload the page after sending the troops, but this was already done at the initial version of the robber_hideouts, so not really sure which was the purpose meant for that. Furthermore, it also was causing problems because it was in the wrong place disturbing actions from other threads.

File: start.py
Comment (pkeweloh) [reply start.py]: README updated with new features!
And I've also adjusted the parameter overload of the robber_hideouts and robber_camps methods so that the optional 'interval' is at the end of the calls.

**Pull request merged**
